pkgs/applications/emulators/retroarch/retroarch-assets.nix

```diff
@@ -0,0 +1,33 @@
+{ lib
+, stdenvNoCC
+, fetchFromGitHub
+}:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "retroarch-assets";
+  version = "unstable-2022-10-24";
+
+  src = fetchFromGitHub {
+    owner = "libretro";
+    repo = "retroarch-assets";
+    rev = "4ec80faf1b5439d1654f407805bb66141b880826";
+    hash = "sha256-j1npVKEknq7hpFr/XfST2GNHI5KnEYjZAM0dw4tMsYk=";
+  };
+
+  makeFlags = [
+    "PREFIX=$(out)"
+    # By default install in $(PREFIX)/share/libretro/assets
+    # that is not in RetroArch's assets path
+    "INSTALLDIR=$(PREFIX)/share/retroarch/assets"
```

Comment:
I don't understand this comment. Is the path mentioned there wrong? Should it match the INSTALLDIR=?

Reply:
The `Makefile` defaults to `$(PREFIX)/share/libretro/assets`: https://github.com/libretro/retroarch-assets/blob/master/Makefile#L2

And yeah, the default path is wrong because RetroArch itself expects it in `$(PREFIX)/share/retroarch/assets`.

Reply:
Wow. Upstream continue to surprise me.